src/components/contextless-android-activity-toggle/contextless-android-activity-toggle.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ToggleableCodeBlock from '../toggleable-code-block/toggleable-code-block';
+import { highlightJava } from '../../helpers/highlight-java';
+import { highlightKotlin } from '../../helpers/highlight-kotlin';
+
+export default class ContextlessAndroidActivityToggle extends React.Component {
+  checkPreference = language => {
+    return this.props.context.preferredLanguage.android === language;
+  };
+
+  render() {
+    const {
+      context,
+      id,
+      java,
+      kotlin,
+      copyRanges,
+      filename,
+      link,
+      limitHeight
+    } = this.props;
+
+    let selectedCode = '';
+    if (context) {
+      selectedCode = this.checkPreference('kotlin') ? kotlin : java;
+    }
+
+    const snippetProps = {
+      copyRanges: copyRanges || undefined,
+      filename: filename || undefined,
+      link: link || undefined,
+      options:
+        kotlin && java
+          ? [
+              {
+                label: 'Java',
+                language: 'java',
+                preferredLanguage: this.checkPreference('java')
+              },
+              {
+                label: 'Kotlin',
+                language: 'kotlin',
+                preferredLanguage: this.checkPreference('kotlin')
+              }
+            ]
+          : undefined
+    };
+
+    return (
+      <div className="my24">
```

Comment:
You'll want to add the prose class here so that the code block can get styles from Mapbox Assembly:

```suggestion
      <div className="my24 prose">
```

Right now the test cases app shows a transparent background with minimal styling on the code blocks:

![image](https://user-images.githubusercontent.com/2180540/64631397-09ec2600-d3c5-11e9-9245-eb33cc119aa9.png)

Reply:
Yes! This was bothering me so much -- I didn't realize `prose` was the solution!